projects/demo/src/modules/components/carousel/examples/4/index.html

```diff
@@ -7,7 +7,7 @@
 </ng-template>
 
 <ng-template
-    [tuiDialogOptions]="{ size: 's', header: header }"
+    [tuiDialogOptions]="$any({ size: 's', header: header })"
```

Comment:
I saw this issue and I'd like to fix it instead of adding `$any`. Let's dig into it.

Reply:
can we leave TODO here for now? and solve it in the next PR?

Reply:
Sure, but this seems important, let's not forget about it.